src/sentry/identity/google/provider.py

```diff
@@ -53,6 +60,7 @@ def build_identity(self, state):
             "email": user_data["email"],
             "email_verified": user_data["email_verified"],
             "name": user_data["email"],
+            "domain": user_data.get("hd", DEFAULT_GOOGLE_DOMAIN),
```

Comment:
Will you use a migration to backfill all the existing google people with a domain, or will you need to be careful ™️ reading this data?

Reply:
There isn't any data yet :)